autoload/clap/cache.vim

```diff
@@ -0,0 +1,54 @@
+" Author: Mark Wu <markplace@gmail.com>
+" Description: Cache API for clap.
+
+let s:save_cpo = &cpoptions
+set cpoptions&vim
+
+let s:path_separator = has('win32') ? '\' : '/'
+let s:clap_cache_dir = get(g:, 'clap_cache_dir', '')
+
+function! clap#cache#folder() abort
```

Comment:
I personally would rename it to `clap#cache#directory` to be more consistent with `g:clap_global_directory`.

Reply:
Done!